rpcd/playbooks/ironic-post-install-config.yml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
This needs to be formatted like a playbook since it's called directly with openstack-ansible (which in turn uses ansible-playbook).